glean-core/src/metrics/timing_distribution.rs

```diff
@@ -247,13 +265,15 @@ impl TimingDistributionMetric {
                 if sample < 0 {
                     num_negative_samples += 1;
                 } else {
-                    let sample = sample as u64;
-                    let mut sample = self.time_unit.as_nanos(sample);
+                    let mut sample = sample as u64;
+
                     if sample > MAX_SAMPLE_TIME {
```

Comment:
No minimum check here?

Reply:
It could still be `0`, which we don't allow for others.

Reply:
Good point